jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/BuildServiceTest.java

```diff
@@ -93,4 +105,186 @@ void tagImage_whenValidImageConfigurationProvided_shouldTagImage() throws Docker
         .tag("image-name", "image-name:latest", true);
   }
 
+  @Test
+  void mergeBuildArgs_whenBuildArgsFromImageConfigAndFromProjectProperties_shouldMergeBuildArgs() {
```

Comment:
maybe all `mergeBuildArgs_*` tests can be moved into a `@Nested` test class? Maybe these tests can share common code in `// Given` block in `@BeforeEach`?

Reply:
I have refactored a bit, let me know if it is better or not